superset-frontend/src/visualizations/FilterBox/FilterBoxChartPlugin.js

```diff
@@ -22,9 +22,12 @@ import thumbnail from './images/thumbnail.png';
 import controlPanel from './controlPanel';
 
 const metadata = new ChartMetadata({
+  category: t('Tools'),
   name: t('Filter box'),
-  description:
-    'A multi filter, multi-choice filter box to make dashboards interactive',
+  description: t(`Chart component that lets you add a custom filter UI in your dashboard. When added to dashboard, a filter box lets users specify specific values or ranges to filter charts by. The charts that each filter box is applied to can be fine tuned as well in the dashboard view.
+
```

Comment:
Just wondering why the extra dropped row here, am I missing something that I should not? Sorry if that's how it is supposed to be and my ignorance is at fault here!

Reply:
It's just there for formatting.

Reply:
Thanks!